introduction.md

```diff
@@ -1 +1,6 @@
 Hello my name is Lukas and I am new to the dojo. My favorite codeschool path is JavaScript.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?